actions/build-ml-whl/action.yaml

```diff
@@ -1,4 +1,4 @@
-name: "Build PyPi"
+name: "Build PyPi Wheel"
```

Comment:
can you update this name dropping the reference to PYPI before merging?

Reply:
updated.